### ISSUE TITLE
Project Metadata documentation has errors

### ISSUE THREAD

**Issue by awhitford** [sbt/sbt#947]
I noticed that there are a couple of bugs with the [Project Metadata](http://www.scala-sbt.org/release/docs/Howto/metadata.html) page:

Specifically, it mentions:

```
organizationHomepage := "org.example"
```

But that doesn't work...  Needs to be something like:

```
organizationHomepage := Some(url("http://example.org"))
```

And also, this doesn't work:

```
licenses += "GPLv2" -> "http://www.gnu.org/licenses/gpl-2.0.html"
```

Needs to be something like:

```
licenses += "GPLv2" -> url("http://www.gnu.org/licenses/gpl-2.0.html")
```

**Comment by harrah** [comment]
Thanks for reporting.  Please feel free to fix by submitting a pull request!

**Comment by awhitford** [comment]
Created pull request:  https://github.com/sbt/sbt/pull/969

**Comment by harrah** [comment]
Fixed in e7d5cd1bd25aa24abb6a1ead3a2701bc451ff724.
